### DISCUSSION TITLE
Toggle --ignore-hosts Option During mitmdump runtime with Python

### DISCUSSION THREAD

**Discussion by montovaneli** [mitmproxy/mitmproxy#6429] [Q&A] [answered]
Hello all,

Is there any way to "turn on" or "turn off" the `--ignore-hosts mywebsite.com` option during the execution of mitmdump using Python, or is it only possible during initialization?

For example, I want to ignore the host during the first two minutes of code execution, but then I want to start intercepting it.

**Comment by mhils** [comment, marked as answer]
You can turn it on/off anytime by setting `ctx.options.ignore_hosts` at runtime. :)

**Reply by montovaneli** [reply]
Thank you so much!
